.eslintignore

```diff
@@ -2,4 +2,5 @@ node_modules/
 .next/
 dist/
 src/prisma/prismaservice/generated/
-src/prisma/prismaservice/dist/
\ No newline at end of file
+src/prisma/prismaservice/dist/
+src/typings/global.d.ts
```

Comment:
O if you look in global.d.ts I used disable no var for one line. Either remove this or ignore the file. Now the ignore is pointless